README.md

```diff
@@ -39,6 +39,17 @@ To setup on your local machine:
 
 **NOTE** - The [Alternating Least Squares (ALS)](notebooks/00_quick_start/als_movielens.ipynb) notebooks require a PySpark environment to run. Please follow the steps in the [setup guide](SETUP.md#dependencies-setup) to run these notebooks in a PySpark environment.
 
+## Install this repository via PIP
+A [setup.py](reco_utils/setup.py) file is provied in order to simplify the installation of this utilities in this repo from the main directory. 
+
+    pip install -e reco_utils
+
+It is also possible to install directly from Github.
+
+    pip install git+http://github.com/microsoft/recommenders/#egg=pkg&subdirectory=reco_utils
+
+**NOTE** - The pip installation does not install any of the necessary package dependencies, it is expected that conda will be used as shown above to setup the environment for the utilities being used.
```

Comment:
Not entirely sure but maybe something like `reco_utils[pyspark, gpu, cpu]` is useful.

Reply:
yep, I'll test that out. will need to pull in requirements from the generate_conda script.